tests/rules/assertions/enforceSuppressionCode.js

```diff
@@ -0,0 +1,90 @@
+export default {
+  invalid: [
+    {
+      code: '// $FlowFixMe I am doing something evil here\nconst text = \'HELLO\';',
+      errors: [
+        {
+          message: '$FlowFixMe is missing a suppression code',
```

Comment:
Flow is warning with the following message when the suppression code is missing:

```text
Suppression is missing a code. Please update this suppression to use an error code: $FlowFixMe[incompatible-cast]
```

I understand that this would be complicated to do in Eslint, but maybe it would be nice to explain how to fix this error? For example:

```text
$FlowFixMe is missing a suppression error code. Please update this suppression to use an error code: $FlowFixMe[…]
```

What do you think? Just a nit.

Reply:
Yeah no worries, let me look into it later

Reply:
Done, agree it's heaps better